erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -13,6 +13,7 @@
     5: "ELSTER_REQUEST_ID_UNKNOWN",
     6: "INVALID_BUFA_NUMBER",
     7: "INVALID_TAX_NUMBER",
+    8: "ALREADY_REVOKED",
```

Comment:
Julian and I said a 4 would be better and we set the error message to "ALREADY_REVOKED_UNLOCK_CODE". As far as I know the 4 hasn't been there for a long time